### PR TITLE
upcoming workshop

### PR DESCRIPTION
**Pull request by jburel**
omero workshop to be added when it is announced

### CODE REVIEW

**Comment by joshmoore** [comment]
@jburel : do you want me to try the headline suggestion here? And if so, @lunson, do you have an idea on how to add an "Upcoming" icon/cutie (:smile:)/etc to clearly mark the newest entries on https://www.openmicroscopy.org/events/.

**Comment by jburel** [comment]
@joshmoore feel free to push to this branch to try it out on. It is now a good example since we have 4 upcoming workshops

**Comment by lunson** [comment]
@joshmoore Simplest way is to use an icon from fontawesome (the CSS is already in the repo). I recommend `star` (https://fontawesome.com/icons/star?style=solid) or a colored `circle` (https://fontawesome.com/icons/circle?style=solid) similar to what we see on phone apps when a new version is downloaded.

**Comment by joshmoore** [comment]
Thanks!

**Comment by joshmoore** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ and main page updated. Stars might be a bit too subtle:
Screen Shot 2019-07-29 at 16 53 27

**Comment by lunson** [comment]
A color will add some pop and get it noticed: `<i class="fa fa-star" style="color: #df283f;"></i>`

**Comment by joshmoore** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ updated with @lunson's color suggestion:

Screen Shot 2019-07-30 at 09 13 44

**Comment by jburel** [comment]
Maybe a big icon will make it even more noticeable
but that might break the "balance" of the page.
It certainly looks nice.
I will review the content of the text today following the adjustment to the announcement so we can deploy it later on

**Comment by lunson** [comment]
The standard resizing for FontAwesome is to use `fa-2x`, `fa-3x`, etc. Attached is what the next size up (`2x`) looks like.
Screen Shot 2019-07-30 at 9 16 25 AM

I recommend we stay put at the default `1x`. It serves its purpose in that it is noticeable without being obnoxiously so, and it distinguishes the upcoming from the past events.

**Comment by jburel** [comment]
Thanks @lunson. Let's go with your suggestion and keep it to the default size

File: events/index.html [hunk 1]
```diff
@@ -8,6 +8,30 @@
         <!-- begin Events -->
         <hr class="whitespace">
         <div id="community-posts" class="row">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> November 11, 2019</h6>
+                <h2><a href="omero-users-workshop-hamilton-november-2019.html" target="_blank">OMERO Workshop, Hamilton</a></h2>
```
Comment (manics) [inline events/index.html]: Is `target="_blank"` needed? Opening a new window seems unnecessary.

File: events/index.html [hunk 2]
```diff
@@ -8,33 +8,57 @@
...
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> October 09-10, 2019</h6>
+                <h2><a href="omero-users-workshop-freising-october-2019.html">OMERO Workshop, Freising</a></h2>
+                <p class="card-caption">A two-day workshop on how to use OMERO.</p>
+            </div>
+            <hr class="medium-8">
+            <div class="small-12 medium-8 medium-offset-2 columns">
+                <h6><i class="fa fa-star" style="color: #df283f;"></i> September 30-October 1, 2019</h6>
```
Comment (mtbc) [inline events/index.html]: we use 09 and 05 above and below so maybe 01 here (if we want the 0-prefixes at all)

File: events/omero-dev-workshop-dundee-september-2019.html [hunk 3]
```diff
@@ -0,0 +1,58 @@
+---
+layout: news-events
+filename: events
+title: April 2019 OMERO users workshop
```
Comment (mtbc) [inline events/omero-dev-workshop-dundee-september-2019.html]: not April I guess

**Comment by dominikl** [comment]
Could we please make it very clear, that these workshops are not organized by OME and there is no public registration (apart from the Dundee one obviously). I just got another email yesterday from someone asking about where to sign up for the Freising workshop.

**Comment by jburel** [comment]
We have a row Organiser 
Maybe a note indicating to contact organiser directly

**Comment by manics** [comment]
Something for the future: it could be worth going down the idr website route of storing event information in a CSV and converting it for display in jekyll, https://github.com/IDR/idr.openmicroscopy.org/blob/master/_data/releases.tsv

**Comment by jburel** [comment]
@dominikl: last commit should address the issue

**Comment by joshmoore** [comment]
Longer-term (after this PR) I could see adding more context than "future" v. "past" to the `/events` page:
 - past
 - future: registration closed/full
 - future: registration open
 - future: registration not-yet open
 - future: external

**Comment by manics** [comment]
The line-spacing mentioned in https://github.com/ome/www.openmicroscopy.org/pull/303#pullrequestreview-268412256 is still present

**Comment by jburel** [comment]
@manics thanks for the clarification. I misunderstood, I though you were talking about the space between the two red boxes since they were almost touching

File: events/omero-users-workshop-freising-october-2019.html [hunk 4]
```diff
@@ -5,6 +5,11 @@
 main-blurb: Two days workshop on how to use OMERO held in Freising, Germany.
 ---          
      <hr class="whitespace">
+        <div class="row">
+            <div class="column">
+                <h2 class="event-day">Registration: Contact directly the organiser</h2>
```
Comment (mtbc) [inline events/omero-users-workshop-freising-october-2019.html]:
```suggestion
                <h2 class="event-day">Registration: Contact the organiser directly</h2>
```

File: css/openmicroscopy.css [hunk 5]
```diff
@@ -275,6 +275,7 @@ h3.subdivider {
     font-weight: 600;
     margin-bottom: 5px;
     text-shadow: 1px 1px 3px rgba(0,0,0,1);
+    line-height: 1rem;
```
Comment (manics) [inline css/openmicroscopy.css]: I think this addition has broken the line spacing on all pages, and not just https://github.com/ome/www.openmicroscopy.org/pull/303#issuecomment-516770392

https://snoopycrimecop.github.io/www.openmicroscopy.org/teams/
Screen Shot 2019-08-01 at 10 55 05

https://www.openmicroscopy.org/teams/
Screen Shot 2019-08-01 at 10 55 10

@jburel If this is the problem then https://github.com/ome/www.openmicroscopy.org/pull/303/commits/b1038151a5ed018b85a4cf53b97b9b390cf0fef7 can be reverted

**Comment by jburel** [comment]
@manics I have used 1.4 in CSS we might want to go to 1.3

**Comment by joshmoore** [comment]
I'd vote for getting this in and deployed now that the dev workshop announcement is out.

**Comment by pwalczysko** [comment]
Looks good. Ready to merge FMPOV.

**Comment by jburel** [comment]
Thanks merging and deploying website

**Pull request merged**